terraform/staging/main.tf

```diff
@@ -135,12 +135,42 @@ module "application_security_group" {
   vpc_id                          = module.networking.vpc_id
 }
 
+moved {

```

Comment:
Plan to remove the moved blocks after PR has been approved and applied to staging (production ones should remain)